File: Modules/Common/Equipments/SIE.Web.Equipments/EquipAccounts/Commands/EquipAccountSlotSaveCommand.js
SIE.defineCommand('SIE.Web.Equipments.EquipAccounts.Commands.EquipAccountSlotSaveCommand', {
    extend: 'SIE.cmd.Save',
    meta: { text: "保存", group: "edit" },
    doSave: function (view) {
        var me = this;

        //此功能父ID丢失，要手动赋值
        var parentId = view.getParent().getCurrent().getId();
        view.getData().data.items.forEach(function (item) {
            item.setEquipAccountId(parentId);
        });

        var children = view.getChildren();
        var withChildren = children.length > 0;
        view.execute({
            withChildren: withChildren,
            success: function (res) {
                me.onSaved(view, res);
            }
        });
    },
});

File: Modules/Common/Equipments/SIE.Web.Equipments/EquipAccounts/Commands/EquipAccountSlotSaveCommand.cs
using SIE.Domain;
using SIE.Web.Command;

namespace SIE.Web.Equipments.EquipAccounts.Commands
{
    /// <summary>
    /// 设备缸槽保存
    /// </summary>
    [JsCommand("SIE.Web.Equipments.EquipAccounts.Commands.EquipAccountSlotSaveCommand")]
    public class EquipAccountSlotSaveCommand : SaveCommand
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="data"></param>
        protected override void DoSave(EntityList data)
        {
            base.DoSave(data);
        }
    }
}
